resources/assets/ckeditor/plugins/filelink/plugin.js

```diff
@@ -0,0 +1,34 @@
+CKEDITOR.plugins.add( 'filelink', {
+    icons: 'filelink',
+    requires: ['iframedialog'],
+    init: function( editor ) {
+        editor.ui.addButton( 'Filelink', {
+            label: 'Insert File',
+            command: 'filelink.cmd',
+            toolbar: 'insert'
+        });
+
+        var cmd = editor.addCommand('filelink.cmd', {exec:showDialogPlugin});
+
+        CKEDITOR.dialog.addIframe(
+            'filelink.dlg',
+            'File Link',
+            editor.config.fileLinkUploadUrl,
+            800,
+            400,
+            function(){
```

Comment:
@ercanozkaya Sir, need your assistance here, can you help with closing the modal dialogue (iframe) when clicking `Insert` button from the loaded FILElink view?

![image](https://user-images.githubusercontent.com/5572579/34149411-eea3f682-e4df-11e7-95fe-f2bc1c171895.png)

Reply:
Hey @jebbdomingo , is there a reason we are opening Ckeditor's modal instead of our own magnificPopup? We have another ticket to use the same modal everywhere.

It's also easy to use like this:
```
kQuery.magnificPopup.open({
                    items: {
                        src: $(this).attr("href"),
                        type: "iframe"
                    },
                    mainClass: "koowa_dialog_modal"
                });```

Can you try to use this?